adagrams/game.py

```diff
@@ -1,11 +1,118 @@
+import random
+import string
 def draw_letters():
```

Comment:
Notice that the `draw_letters` method has two responsibilities. It is creating a letter pool to draw from and then drawing random letters. To keep your methods to a single responsibility, consider pulling out this logic into a helper function and then invoking that helper function in draw_letters. Maybe something like `create_letter_pool`?